.env

```
@@ -0,0 +1,2 @@
+export DEV_DATABASE=alpha-blog-dev
```

you don't need to add `export` here.
```suggestion
DEV_DATABASE=alpha-blog-dev
```

---

"You may also add export in front of each line so you can source the file in bash:"
I think it sets the variable in the bash file, i didn't checked it

---

That means if you want to, but in your case you are not sourcing the file